server/middleware/validInfo.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function(req, res, next) {
+   const { firstName, lastName, userRole, userEmail, userSlack, userPassword, userGithub, userClassId ,username,userPhone,cyfCity,userDateOfBirth} = req.body;
+
+  function validEmail(userEmail) {
+    return /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(userEmail);
+  }
+
+  if (req.path === "/register") {
+    console.log(!userEmail.length);
+    if (![userEmail, username, userPassword].every(Boolean)) {
```

Comment:
I didn't know about `Array.every()`! Learn something new every day :)

Reply:
ohh  this is because we covered array in great detail at cyf.